### DISCUSSION TITLE
Can I send a stored text PLUS the [ENTER] signal?

### DISCUSSION THREAD

**Discussion by lordsnowbr** [hluk/CopyQ#1940] [Q&A]
Hi, 
I'd like to be able to send a text stored and after the text pasted, send a {ENTER] keyboard stroke. Is that currently possible? 
Thanks

**Comment by hluk** [comment]
Generally, sending keystrokes is not supported by CopyQ. The feature is out of scope for this application but you could map a shortcut to trigger a command that uses another utility (like `xdotool` that can generate the key events).
